css/types/image.json

```diff
@@ -405,6 +405,306 @@
                 }
               }
             }
+          },
+          "repeating-linear-gradient": {
+            "__compat": {
+              "description": "<code>repeating-linear-gradient()</code>",
+              "support": {
+                "webview_android": {
+                  "version_added": true
+                },
+                "chrome": [
+                  {
+                    "version_added": "26"
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "10"
+                  }
+                ],
+                "chrome_android": {
+                  "version_added": true
```

Comment:
webkit prefix

Reply:
Good catch. I also fixed it for linear-gradient()